src/sqlfluff/core/rules/std/L015.py

```diff
@@ -1,8 +1,10 @@
 """Implementation of Rule L015."""
 
 from ..base import BaseCrawler, LintFix, LintResult
+from ..doc_decorators import document_fix_compatible
 
 
+@document_fix_compatible
```

Comment:
This is not related to the main PR -- it corrects an oversight from my earlier PR (#732).